src/Core/x/xstransvc.cpp

```diff
@@ -1,12 +1,36 @@
 #include "xstransvc.h"
 
 #include <types.h>
+#include <string.h>
+
+extern void XST_unlock_all();
+extern int32 PKRShutdown();
+extern int8* xST_xAssetID_HIPFullPath(uint32 aid, uint32* sceneID);
+extern int XST_cnt_locked();
+extern int XST_nth_locked(int param);
```

Comment:
Put these function definitions in their appropriate header files. You also shouldn't need the `extern` keyword.